lighthouse-cli/test/global-mocha-hooks.js

```diff
@@ -1,3 +1,4 @@
+// @ts-nocheck
```

Comment:
I don't think we've used this and `lighthouse-core/test/global-mocha-hooks` for long time.

Reply:
Neat, deleted.

Reply:
oops, pushed.